Makefile

```
@@ -350,3 +350,13 @@ release-changelog: $(wildcard changelog/*.txt)
 	@:$(if $(LAST_RELEASE),,$(error please set the LAST_RELEASE environment variable for changelog generation))
 	@:$(if $(THIS_RELEASE),,$(error please set the THIS_RELEASE environment variable for changelog generation))
 	changelog-build -changelog-template changelog/changelog.tmpl -entries-dir changelog -git-dir . -note-template changelog/note.tmpl -last-release $(LAST_RELEASE) -this-release $(THIS_RELEASE)
+
+.PHONY: dev-gorelease
+dev-gorelease: export GORELEASER_PREVIOUS_TAG := $(shell git describe --tags $(shell git rev-list --tags --max-count=1))
```

not sure if this behaves like we want it to? On `main` this currently shows `v2.0.2` as the previous tag, even though this tag was created on the `release/v2.0.x` branch. I'd suggest to use this command instead to find the last tag in the parent of the current branch
```suggestion
dev-gorelease: export GORELEASER_PREVIOUS_TAG := $(shell git describe --tags --exclude "api/*" --exclude "sdk/*" --abbrev=0)
```